renderer/src/GLRenderer.cpp

```diff
@@ -380,7 +385,7 @@ void GLRendererImplementation::renderFrame() {
     initGL();
 
     for (const auto &[id, object] : objects_) {
-        const auto material_component = object.getComponent("material");
+        const auto material_component = object.getComponent<rendering::Material>();
```

Comment:
Copy? You sure?

Reply:
Yup, the method returns a copy. Left a `TODO` to optimize using const ref.

Reply:
'Kay